### PR TITLE
Data Tab redundant - remove

### PR DESCRIPTION
**Pull request by RossKen**
Even if this tab would be useful for seeing all the data, it is really slow in Shiny. An alternative would be an option to download the raw data as a csv so users can have all the data in one place. However, this has not been asked for in user testing so could be giving ourselves extra work needlessly

### CODE REVIEW

**Comment by EdwardAndress** [comment]
Good call Ross.  If there is no clear user need we should probably hold off providing the download function.

File: server.R [hunk 1]
```diff
@@ -200,37 +200,4 @@ function(input, output) {
...
-                           "Topic Keywords"),
-              filter = "top",
-              options = list(scroller = TRUE,
-                             paging = FALSE
-                             #autoWidth = TRUE,
-                             #columnDefs = list(list(width = "30%", targets = list(2,3)))
-                             ))
-  })
-}
+}
```
Comment (EdwardAndress) [inline server.R]: There should be a new line here.  It's just standard practice and will likely get caught by the linter.  Having a new line at the end of the file also improves the way in which diffs are represented in Git.

Comment (RossKen) [inline server.R]: Did not know that was good practice, noted! (and I will go fix)

**Comment by RossKen** [comment]
@samtazzyman we have a branch for the UserTest version, so this should not be an issue

**Pull request merged**